docs/compatibility.md

```diff
@@ -49,10 +49,12 @@ Distrobox has been successfully tested on:
 | EndlessOS | 4.0.0 | |
 | OpenSUSE | Leap 15<br>Tumbleweed | |
 | OpenSUSE MicroOS | 20211209 | |
-| Void Linux | glibc | Systemd service export will not work.<br>To setup graphical app exporting, you need to install `xhost` and add this to your `~/.xinitrc`: <br> `xhost +si:localuser:$USER`.  |
+| Void Linux | glibc | Systemd service export will not work. |
 | NixOS | 21.11 | __NOTE NixOS support is preliminary, and there are many bugs present, any help in improving support is appreciated__ <br> Currently you must have your default shell set to Bash, if it is not, make sure you edit your configuration.nix so that it is. <br> To install distrobox:<br>`mkdir -p ~/.local/bin`<br>Add `PATH=$PATH:$HOME/.local/bin` to your bashrc<br>Execute [THIS](#installation) command without sudo.<br>To setup Docker, look [HERE](https://nixos.wiki/wiki/Docker) <br>To setup Podman, look [HERE](https://nixos.wiki/wiki/Podman) and [HERE](https://gist.github.com/adisbladis/187204cb772800489ee3dac4acdd9947) |
 | Windows WSL2 | | __NOTE WSL2 support is preliminary, and there are many bugs present, any help in improving support is appreciated__ <br> Currently you must work around some incompatibility between WSL2 and Podman, namely [THIS](https://github.com/containers/podman/issues/12236). <br>Install into WSL2 any of the supported distributions in this list. <br> Ensure you have an entry in the `fstab` for the `/tmp` folder:<br> `echo 'tmpfs /tmp tmps defaults 0 0' >> /etc/fstab`.<br>Then reboot the WSL machine `wsl --shutdown` <br>Note that `distrobox export` is not supported on WSL2 and will not work. |
 
+If your container is not able to connect to your host xserver, make sure to install `xhost` on the host machine and run `xhost +si:localuser:$USER` or add it to your `~/.xinitrc`.
```

Comment:
Should Be **and** add it to xinirc to make it permanent

Reply:
The goal is to make it run on every x session startup. Editing `.xinitrc` may not affect it if you're, let's say, using KDE Plasma but your `.xinitrc` starts Gnome.